tools/topology/topology2/doc/README

```diff
@@ -0,0 +1,8 @@
+To build the topology2 source documentation do following steps:
+
+cd tools/topology/topology2/doc
+mkdir build
+cd build
+cmake ..
```

Comment:
This is very convoluted way to use CMake. Replace everything with these two lines:
```suggestion
cmake -S tools/topology/topology2/doc -B build_dir1
cmake --build build_dir1

```